iron-swipeable-container.html

```diff
@@ -0,0 +1,268 @@
+      cursor: default;
+    }
+
+@group Iron Elements
+@element iron-swipeable-container
+@demo demo/index.html
+-->
+
+<dom-module id="iron-swipeable-container">
+  <template>
+    <content id="content"></content>
+  </template>
+</dom-module>
+
+<script>
+
+  Polymer({
+    is: 'iron-swipeable-container',
+
+    /**
+     * Fired when a child element is swiped away.
+     *
+     * @event iron-swipe-away
+     */
+
+    properties: {
+      /**
+       * The style in which to swipe the card. Currently supported
+       * options are `curve | horizontal`. If left unspecified, the default
+       * is assumed to be `curve`.
+       */
+      swipeStyle: {
+       type: String,
+       value: 'curve'
+      },
+
+      /**
+       * If true, then the container will not allow swiping.
+       */
+      disableSwipe: {
+        type: Boolean,
+        value: false
+      },
+
+      /**
+       * The ratio of the width of the element that the translation animation
+       * should happen over. For example, if the `widthRatio` is 3, the
+       * animation will take place on a distance 3 times the width of then
+       * element being swiped.
+       */
+      widthRatio: {
+       type: Number,
+       value: 3
+      },
+
+      /**
+       * The ratio of the total animation distance after which the opacity
+       * transformation begins. For example, if the `widthRatio` is 1 and
+       * the `opacityRate` is 0.5, then the element needs to travel half its
+       * width before its opacity starts decreasing.
+       */
+      opacityRate: {
+       type: Number,
+       value: 0.2
+      },
+
+      /**
+       * The CSS transition applied while swiping.
+       */
+      transition: {
+        type: String,
+        value: '300ms cubic-bezier(0.4, 0.0, 0.2, 1)'
+      },
+
+      /**
+       * The CSS transition property applied while swiping.
+       */
+      transitionProperty: {
+        type: String,
+        value: 'opacity, transform'
+      }
+    },
+
+    listeners: {
+      'iron-swipe-away': '_swipeAway'
+    },
+
+    attached: function() {
+      this._nodeObserver = Polymer.dom(this.$.content).observeNodes(
+          function(mutations) {
+            for (var i = 0; i < mutations.addedNodes.length; i++) {
+              this._addListeners(mutations.addedNodes[i]);
+            }
+            for (var i = 0; i < mutations.removedNodes.length; i++) {
+              this._removeListeners(mutations.removedNodes[i]);
+            }
+          }.bind(this));
+    },
+
+    _addListeners: function(node) {
+      if (node.nodeType === Node.TEXT_NODE)
+        return;
+      // Set up the animation.
+      node.style.transitionProperty = this.transitionProperty;
+      node.style.transition = node.style.webkitTransition = this.transition;
+
+      this.listen(node, 'track', '_onTrack');
+      this.listen(node, 'transitionend', '_onTransitionEnd');
+      this.listen(node, 'webkitTransitionEnd', '_onTransitionEnd');
```

Comment:
We don't need `webkit` prefix for the `transition`.  All the supporting browsers should work fine without the prefix.  So we can remove this line and all the other `webkitTransition` stuffs.